orgs/schema-validation.json

```diff
@@ -75,6 +75,27 @@
             "supportUrl"
           ],
           "type": "object"
+        },
+        "journeyAPIConfigurations": {
+          "additionalProperties": false,
+          "properties": {
+            "WAIT_RELUCTANCE": {
```

Comment:
Do these need to be capitalised when they're not defined as constants, but as properties in an object?